tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py

```diff
@@ -89,7 +87,9 @@ def test_upgrade_topology_with_named_repartition_topic(self):
 
     @staticmethod
     def verify_running(processor, message):
-        with processor.node.account.monitor_log(processor.STDOUT_FILE) as monitor:
+        node = processor.node
+        with node.account.monitor_log(processor.STDOUT_FILE) as monitor:
+            processor.start()
```

Comment:
This is the other part of the fix moved processor start here

Reply:
What's the significance of after opening the monitor? Does it block on the log file somehow?

Reply:
This closes a very small timing window, I've made similar changes in other system tests before

Reply:
@vvcephei If get don't get the monitor first, we might miss some output lines from the file in `wait_until` below and thus time out even if the correct output was written.